### DISCUSSION TITLE
Suggestion: Perhaps a Disclaimer: Learn TailwindCSS first.

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1135] [Ideas / feature requests]
Hi there

This is just a suggestion, based on a grave mistake I am realizing I made.

DISCLAIMER:

**Learn TailwindCSS first**

Why I am saying this? When I started learning daisyUI during the week-end, I skimmed though the documentation just to establish the flow and what was happening. I have never attempted to learn TailwindCSS before, grave mistake. Some things were easy to replicate and others... well. Understandable anyway.

I am now realizing that your documents assume that people know very well what they are doing and perhaps are web ninjas, which is not always the case and without a proper understanding of how TailwindCSS works first, as a newbie, one should do themselves justice and a favour. Only come to learn how daisy work after they have a good grasp of TailwindCSS.

Overall, the daisyUI experience has been lovely. I'm trying to reproduce some stuff I got to do with Vuetify, just for relevance anyway on my part.

**Comment by saadeghi** [comment]
I'm moving this topic to Discussions.

I should mention a few things here:

1.daisyUI is titled as a `Tailwind CSS component library` so I think it's clear enough that it's an addition to Tailwind CSS.
2. It is possible to use daisyUI without Tailwind CSS (using other utility first libraries, or using custom CSS for layouts and positioning) but using Tailwind CSS would be easier because it's more mature than other utility first libraries.
3. daisyUI only provides components. It doesn't give you CSS for flexbox, sizing, fonts, etc. But that doesn't mean you should necessarily learn Tailwind CSS. You can handle those styles with your own custom CSS or other libraries.
4. In my opinion, learning the basics of Tailwind CSS for essential use, is easier than most people assume.
